### DISCUSSION TITLE
如何实现子分类？

### DISCUSSION THREAD

**Discussion by AuntYang** [Qexo/Qexo#342] [Q&A]
Qexo文章编辑页的侧边栏提供的分类参数在添加多个分类时，将categories的多个分类解析为并列分类，如何实现子分类呢？如何实现hexo官方文档中多样的分类方式？
![image](https://user-images.githubusercontent.com/49216233/229674029-2f1bdab5-2789-4022-9788-d8069be8a446.png)

**Comment by am-abudu** [comment]
目前还没有办法，因为我也没有想到适当的逻辑。

你只能自定义侧边栏，把分类项目去掉

**Reply by AuntYang** [reply]
好的好的，我在其他参数里设置categories能够实现子分类
